client/my-sites/importer/uploading-pane.jsx

```diff
@@ -113,7 +113,22 @@ class UploadingPane extends React.PureComponent {
 	startUpload = file => {
 		const { startUpload } = this.props;
 
-		startUpload( this.props.importerStatus, file );
+		if ( window.chrome ) {
```

Comment:
This is also truthy on Edge - looks like you could check for

```javascript
window.navigator.vendor === "Google Inc."
```

which is `""` on Edge. 

https://stackoverflow.com/questions/4565112/javascript-how-to-find-out-if-the-user-browser-is-chrome

Reply:
This condition should work, `chrome.webstore` is undefined on Edge:

```javascript
window.chrome && window.chrome.webstore
```